### PR TITLE
feat(dop): pipeline guides list optimization

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What this PR does / why we need it:
Pipeline guides list optimization.

## I have checked the following points:
- [x] I18n is finished and updated by cli
- [x] Form fields validation is added and length is limited
- [x] Display normally on small screen
- [x] Display normally when some data is empty or null
- [x] Display normally in english mode

## Which issue(s) this PR fixes:
Fixes #

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=305231&issueFilter__urlQuery=eyJzdGF0ZXMiOls0NDAyLDcxMDQsNzEwNSw0NDAzLDQ0MDQsNzEwNiw0NDA2LDQ0MDcsNDQxMiw0NTM4LDQ0MTMsNDQxNCw0NDE1LDQ0MTZdfQ%3D%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&iterationID=1174&type=TASK)

## Does this PR introduce a user interface change?

✅ Yes(screenshot is required)
image

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English | Pipeline notification adds a list of files and supports one-click pipeline creation. |
| 🇨🇳 中文    | 流水线通知增加文件列表，并支持一键创建流水线。 |

## Need cherry-pick to release versions?
❎ No

### CODE REVIEW

File: shell/app/locales/zh.json [hunk 1]
```diff
@@ -946,6 +946,8 @@
     "Activity": "活动",
     "actual value": "实际值",
     "Add": "新增",
+    "Add all": "全部创建",
+    "Add-create": "创建",
```
Comment (McDaddy) [inline shell/app/locales/zh.json:950]: is this translation from PD?

Comment (hujiahao-hjh) [reply shell/app/locales/zh.json:950]: Yes

File: shell/app/modules/project/common/components/pipeline-new/guide.tsx [hunk 2]
```diff
@@ -34,11 +34,22 @@ interface GuideItem {
   timeCreated: string;
 }
 
+const trigger = (_list: string[], id: string) => {
```
Comment (McDaddy) [inline shell/app/modules/project/common/components/pipeline-new/guide.tsx:37]: I dont understand what's the meaning of `trigger`

File: shell/app/modules/project/common/components/pipeline-new/index.tsx [hunk 3]
```diff
@@ -56,9 +60,49 @@ const Pipeline = () => {
...
+      ref: branch,
+      pipelineYmls: ymls,
+    };
+
+    setLoading(true);
+
+    oneClickCreatePipeline(params)
+      .then((res) => {
+        const { success, data } = res;
+        const { errMsg } = (data || {}) as { errMsg: string };
```
Comment (McDaddy) [inline shell/app/modules/project/common/components/pipeline-new/index.tsx:77]: `res` type seem is wrong

**Review by McDaddy — APPROVED** [review]

**Pull request merged**